DEPRECATIONS.rst

```diff
@@ -0,0 +1,101 @@
+Deprecations: how to migrate
+============================
+
+New Tracer interface
+--------------------
+
+The following classes and methods are deprecated and should not be used.
+Please use the Thread interface instead.
```

Comment:
There's a number of references to `Thread`, which I think are relics of a prior iteration

Reply:
Lol I kept writing Thread rather than Tracer everywhere